### DISCUSSION TITLE
Whether there have other service lifecycle events more than document given 4?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [moleculerjs/moleculer#1195] [Q&A] [answered]
When I read this [service lifecycle events](https://moleculer.services/docs/0.14/services.html#Lifecycle-Events):

there gives 4 lifecycle events:
```
    created() {
        // Fired when the service instance created (with `broker.loadService` or `broker.createService`)
    },

    merged() {
        // Fired after the service schemas merged and before the service instance created
    },
    
    async started() {
        // Fired when broker starts this service (in `broker.start()`)
    }

    async stopped() {
        // Fired when broker stops this service (in `broker.stop()`)
    }
```

I want to know whether there have other service lifecycle events.

because I saw `async afterConnected()` in molecumer-examples/blog demo project.

**Comment by ColinFrick** [comment, marked as answer]
`afterConnected` event is actually part of moleculer-db

https://moleculer.services/docs/0.14/moleculer-db.html#Lifecycle-entity-events
